### ISSUE TITLE
Discord login not working

### ISSUE THREAD

**Issue by GeoZzNL** [Themayu/discord-server-list#4]
The discord login button is not working, we can't do anything with this project, correct?

**Comment by Themayu** [comment]
Indeed. This was a project for a client-side development module I was taking, so unfortunately it hasn't had most of the functionality of a proper server list implemented. I'm thinking of returning to this soon and turning it into a proper website; until then, I don't have the time to maintain it all too much.
